executor.c

```diff
@@ -10,11 +10,13 @@
 
 int executor(char **args, char **env)
 {
-	int success = 0;
-	
-	success = execve(args[0], args, env);
-	if (success < 0)
-		return (0);
+	pid_t childp = fork();
+
+	if (childp != 0)
+		wait(&childp);
```

Comment:
also u don't need to pass childdp to wait, define a new variable `int status` and pass it, wait will change `status` according to termination of child, since we only have 1 child process, we just need to use `wait` without passing the child pid, `waitpid` is the system call that wait for a specific pid